commit 30fef8826452a28d4bcdaff529c3570b13ab280d
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2019-05-06T09:48:20-07:00 (5 months ago)
    
    MANTA-4249 missing default option for instr_upload_min_batch_size breaks upgrade
    Reviewed by: Robert Bogart <Robert.bogart@joyent.com>
    Approved by: Robert Bogart <Robert.bogart@joyent.com>
    Approved by: Richard Kiene <richard.kiene@joyent.com>

diff --git a/README.md b/README.md
index e4f8126..971c1a8 100644
--- a/README.md
+++ b/README.md
@@ -174,7 +174,7 @@ accel-gc` attempts to help with these decisions -- see
   (lines) to include per instruction object.  This ensures that instruction
   files can never have less than a pre-configured number of lines.  This is
   important because performance on the mako side will deteriorate with
-  numerous, small instruction files to process.
+  numerous, small instruction files to process. (default: 1)
 * `GC_INSTR_UPLOAD_BATCH_SIZE` - The maximum number of delete instructions
   (lines) to include per instruction object.  Note that there is no guarantee
   that all instruction files will reach this size.
diff --git a/package.json b/package.json
index afcf0f8..2958ff7 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "manta-garbage-collector",
   "description": "Manta Garbage Collector",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/joyent/manta-garbage-collector.git"
diff --git a/sapi_manifests/manta-garbage-collector/template b/sapi_manifests/manta-garbage-collector/template
index 2aa89df..88ff60b 100644
--- a/sapi_manifests/manta-garbage-collector/template
+++ b/sapi_manifests/manta-garbage-collector/template
@@ -51,7 +51,7 @@
 		}{{^last}},{{/last}}{{/ACCOUNTS_SNAPLINKS_DISABLED}}
 	],
 	"tunables": {
-		"instr_upload_min_batch_size": {{GC_INSTR_UPLOAD_MIN_BATCH_SIZE}},
+		"instr_upload_min_batch_size": {{#GC_INSTR_UPLOAD_MIN_BATCH_SIZE}}{{GC_INSTR_UPLOAD_MIN_BATCH_SIZE}}{{/GC_INSTR_UPLOAD_MIN_BATCH_SIZE}}{{^GC_INSTR_UPLOAD_MIN_BATCH_SIZE}}1{{/GC_INSTR_UPLOAD_MIN_BATCH_SIZE}},
 		"instr_upload_batch_size": {{GC_INSTR_UPLOAD_BATCH_SIZE}},
 		"instr_upload_flush_delay": {{GC_INSTR_UPLOAD_FLUSH_DELAY}},
 		"instr_upload_path_prefix": "{{GC_INSTR_UPLOAD_PATH_PREFIX}}",
